config/solr.yml

```diff
@@ -4,4 +4,4 @@ development:
 test:
   url: <%= ENV['SOLR_TEST_URL'] || 'http://localhost:8985/solr/hydra-test' %>
 production:
-  url: http://your.production.server:8080/bl_solr/core0
+  url: <%= ENV['SOLR_PROD_URL'] || 'http://your.production.server:8080/bl_solr/core0' %>
```

Comment:
Is having a different ENV VAR for each environment needed (e.g `SOLR_<ENVIRONMENT>_URL`)? Can a single ENV VAR of `SOLR_URL` be used? 
Looking at Hyku they do this: https://github.com/projecthydra-labs/hyku/blob/master/config/solr.yml
 and Plum does this: https://github.com/pulibrary/plum/blob/master/config/solr.yml